Formula/qbs.rb

```diff
@@ -4,6 +4,7 @@ class Qbs < Formula
   url "https://download.qt.io/official_releases/qbs/1.20.0/qbs-src-1.20.0.tar.gz"
   sha256 "44961a4bb61580ae821aaf25ebb5a5737bd8fb79ec0474aa2592cdd45cc5171f"
   license :cannot_represent
+  revision 1
```

Comment:
Does using CMake have any user-visible changes? If it doesn't, we can omit the `revision`.

Reply:
done, thanks